Files/View Models/ItemViewModel.cs

```diff
@@ -863,6 +880,16 @@ public async Task EnumerateItemsFromStandardFolder(string path)
 
         }
 
+        private async Task<bool> CheckBitlockerStatus(StorageFolder rootFolder)
+        {
+            if (Path.IsPathRooted(WorkingDirectory) && Path.GetPathRoot(WorkingDirectory) == WorkingDirectory)
+            {
+                IDictionary<string, object> extraProperties = await rootFolder.Properties.RetrievePropertiesAsync(new string[] { "System.Volume.BitLockerProtection" });
+                return (int?)extraProperties["System.Volume.BitLockerProtection"] == 6; // Drive is bitlocker protected and locked
```

Comment:
Are you sure about 6 means drive is protected? Che this [link](https://stackoverflow.com/a/41310139/12258600).

Reply:
Yes that's the link that helped me implement this. I tested it on my pc on a bitlocker drive and the status is 6 when locked, 1 when unlocked and 2 when not protected. If someone else could test it too I would feel more confident.